pkg/cloudprovider/providers/azure/azure_managedDiskController.go

```diff
@@ -100,10 +107,16 @@ func (c *ManagedDiskController) CreateManagedDisk(diskName string, storageAccoun
 //DeleteManagedDisk : delete managed disk
 func (c *ManagedDiskController) DeleteManagedDisk(diskURI string) error {
 	diskName := path.Base(diskURI)
+	resourceGroup, err := getResourceGroupFromDiskURI(diskURI)
+	if err != nil {
+		glog.Errorf("azureDisk - getResourceGroupFromDiskURI(%s) returned error: %v", diskURI, err)
+		resourceGroup = c.common.resourceGroup
```

Comment:
Shouldn't return error here? e.g. if there are disks with same name in different rg, this may delete the wrong disk?

Reply:
correct, I have incorrect thoughts here, worried about data disk URI format change in the future...